src/resource.rs

```diff
@@ -198,7 +198,7 @@ impl OpDef {
             resource: Default::default(), // Currently overwritten when OpDef added to Resource
             name,
             description,
-            args,
+            params: args,
```

Comment:
Rename the fn parameter. (and same thing for `new_with_yaml_types`)

Reply:
yeah just did that